### ISSUE TITLE
A11y_VscodeWebviewUiToolkit_AllComponents_Checkbox_ScreenReader: The nvda does not announces the associated label for any checkboxes when navigating using tab key

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/vscode-webview-ui-toolkit#180]
### Actual:
Issue1: The nvda does not announces the associated label for any checkboxes when navigating using tab key. The label is not provided in the aria-labelledby

Issue2: The name property is also not given to the checkboxes

Similar issue observed with jaws.

### Similar issues observed with below scenarios:

1. Combo box
2. Dropdown
3. Number Field
4. Radio Group
5. Text Area(Default text area)
6. Text Field(Default text field)

### Expected:
The nvda should be announcing the associated label for all checkboxes when navigating using tab key. The label should be provided in the aria-labelledby. Also the name property should be provided  for all the checkboxes.

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_VscodeWebviewUiToolkit_Aug2021;#Webview UI Toolkit for Visual Studio Code;#Win32;#FTP;#DesktopApp;#WCAG1.3.1;#NVDA;#Jaws;#STP;#AINotInScope;

### Environment Details:
Application Name: Vscode Webview Ui Toolkit
Windows Version: Win10
Screen Reader: NVDA(2021.1), Jaws(2021.2102.34)

### Repro Steps:

1. Launch VS Code
2. Open WebView UI Toolkit: All Component extension
3. Tab to Checkbox element and tab to first checkbox
4. Start NVDA and check the announcement

### User Impact:
Screen Reader users will not get the proper information about the check boxes

[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”
